### ISSUE TITLE
Hide labels for input boxes

### ISSUE THREAD

**Issue by sulthan-ahmed** [UKHomeOffice/hof#39]
As part of my project for global entry we have a design of hiding the label for some of the input boxes.  I appreciate it isn't ideal for accessibility but this should be available especially if the PO and Designer insist on having it as part of the Acceptance criteria of a story

**Comment by JoeChapman** [comment]
@sulthan-ahmed  Can this just be done in your custom/application's stylesheet?

**Comment by sulthan-ahmed** [comment]
we managed to do something with the label in the end as suggested with the custom/application stylesheet.  So I suggest closing it. Thank you
